### PR TITLE
Sri Dasam Granth - Mehron Corrections for Issues 1394 to 1479

### PR DESCRIPTION
**Pull request by bhajneet**
Aside from partial completion of issue #1450, everything should be done corrections wise for issues 1394 to 1479.

I have not gone through and matched page numbers for lines. Let me know if I should do that, it's in issue #1575 for tracking.

Build breaks because of gurmukhi-utils translit issue

### CODE REVIEW

File: data/Sri Dasam Granth/0155.json [hunk 1]
```diff
@@ -1999,6 +1999,31 @@
...
+        "translations": {
+          "English": {
+            "Dr. Surinder Singh Kohli": {
+              "translation": "BY THY GRACE CHAUPAI",
+              "additional_information": {}
+            }
+          },
+          "Punjabi": {
+            "Dr. Rattan Singh Jaggi": {
+              "translation": "ਤੇਰੀ ਕ੍ਰਿਪਾ ਨਾਲ: ਚੌਪਈ:",
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0155.json:2021]: Ditto. `ਚੌਪਈ:`

File: data/Sri Dasam Granth/0155.json [hunk 2]
```diff
@@ -1999,6 +1999,31 @@
...
+        "pronunciation": null,
+        "pronunciation_information": null,
+        "type": "Sirlekh",
+        "gurmukhi": {
+          "Budha Dal Mehron": "caupeI ]"
+        },
+        "translations": {
+          "English": {
+            "Dr. Surinder Singh Kohli": {
+              "translation": "BY THY GRACE CHAUPAI",
```
[the remark this answers is not in the document]
Comment (bhajneet) [reply data/Sri Dasam Granth/0155.json:2015]: Can you confirm with an image?

[the remark this answers is not in the document]
Comment (Sarabveer) [reply data/Sri Dasam Granth/0155.json:2015]: `BY THY GRACE` is for `ਤ੍ਵ ਪ੍ਰਸਾਦਿ`, which isn't in the line.

**Comment by Sarabveer** [comment]
Also merge will be blocked as Open Yaiyha ASCII chars are going to change https://github.com/GurbaniNow/gurmukhi-fonts/issues/11

**Comment by bhajneet** [comment]
We can merge now. And when https://github.com/GurbaniNow/gurmukhi-fonts/issues/11 is completed we'll just do a new merge in addition to using new fonts in desktop.

**Pull request merged**